index.js

```diff
@@ -554,3 +561,14 @@ function destroyStream (stream) {
     else if (typeof stream.emit === 'function') stream.emit('close')
   }
 }
+
+function normalizeGlobalLabels (labels) {
+  const result = {}
+  if (!labels) return result
```

Comment:
If you move this up as the first line and return undefined, then it will not take up room on the metadata payload

Reply:
fixed